app/views/groups/edit.html.erb

```diff
@@ -70,26 +70,28 @@
         >
           <%= t(:".advanced.title") %>
         </h2>
-        <div
-          class="pt-4"
-          aria-labelledby="settings-delete-heading"
-        >
-          <h3 class="text-lg font-semibold leading-tight text-red-600">
-            <%= t(:"groups.edit.advanced.delete_group.title") %>
-          </h3>
-          <p class="text-gray-600 dark:text-gray-400">
-            <%= t(:"groups.edit.advanced.delete_group.description") %>
-          </p>
-          <div class="mt-4">
-            <%= link_to t(:"groups.edit.advanced.delete_group.submit"),
-                        group_path(@group),
-                        data: {
-                          turbo_method: :delete,
-                          turbo_confirm: t(:"groups.edit.advanced.delete_group.confirm")
-                        },
-                        class: "btn btn-danger" %>
+        <% if allowed_to?(:allowed_to_destroy?, @group) %>
```

Comment:
If you are not "allowed to destroy" the user will see and empty  Advanced setting card (these will need to be updated to the new card component but not in this PR), wouldn't it make more sense to hide the advanced settings card altogether?

Reply:
Also, I would move the advanced settings out into it's own partial.

Reply:
Updated to hide advanced settings card if the member cannot destroy the group in 02c67837cf02c47a85cd379f32009266d612df22.

Moving the advanced settings out into it's own partial will be done in another PR

Reply:
Thanks :)